src/clj_puppetdb/core.clj

```diff
@@ -19,6 +19,24 @@
   ([^String host] (connect host {}))
   ([^String host opts] (http/make-client host opts)))
 
+(defn- encode-order-by-using-keyword
+  "order-by params contain nested maps and if the VCR running we want to enforce a specific ordering to give us URL stability"
+  [params client keyword]
+  (if (contains? params keyword)
+    (update-in params [keyword]
+      (if (get-in client [:opts :vcr-dir])
+        ; Stability of URL is important
+        (fn [order-by] json/encode (map (fn [map] into (sorted-map) map) order-by))
```

Comment:
It seems that some brackets are missing on line 28 - " json/encode" and "into" seem to be odd now.